.azure-pipelines/steps/install-dotnet-sdk.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+  - name: sdkVersion
+    type: string
+    default: ''
+
+  - name: channel
+    type: string
+    default: ''
+
+steps:
```

Comment:
This file doesn't appear to be used? 🤔

Reply:
Removed in https://github.com/DataDog/dd-trace-dotnet/commit/242c3c813e05991fdf1543aa6f7574ce577c6805